agent/requestor-sdk/examples/run_tasks.rs

```diff
@@ -20,9 +20,9 @@ async fn main() -> Result<()> {
     ])
     .with_tasks(vec!["1", "2"].into_iter().map(|i| {
         commands! {
-            upload(format!("input-{}.txt", i), "/workdir/input.txt");
+            upload(format!("input-{}.txt", i).into(), "/workdir/input.txt");
```

Comment:
@filipgolem these should not be required anymore, as they are auto-appended in the macro itself. Did it not work for you without those?